docker-build-package

```diff
@@ -18,9 +18,10 @@ set -e
 
 IMAGE_NAME_PREFIX=registry.gitlab.com/northern.tech/mender/mender-test-containers:mender-dist-packages-builder
 
+# TODO (MEN-6569) - hardcoded to lluiscampos whilst this is a feature branch
 declare -A mender_client_props=(
     [recipe_name]="mender-client"
-    [src_url]="https://github.com/mendersoftware/mender"
+    [src_url]="https://github.com/lluiscampos/mender"
```

Comment:
This we need to amend before the merge and at the same time as we merge `mender`'s feature branch. I will amend.

Reply:
This will be done shortly after the merge.

Reply:
See:
* https://github.com/mendersoftware/mender-dist-packages/pull/385